File: beginner/javascript/1074.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split('\n');

var n, i, x;

n = parseInt(valores.shift());

for(i = 0; i < n; i++){
    x = parseInt(valores.shift());

    if(x == 0)
        console.log("NULL");
    else if(x > 0 && x % 2 == 0)
        console.log("EVEN POSITIVE");
    else if(x > 0 && x % 2 != 0)
        console.log("ODD POSITIVE");
    else if(x < 0 && x % 2 == 0)
        console.log("EVEN NEGATIVE");
    else if(x < 0 && x % 2 != 0)
        console.log("ODD NEGATIVE");
}

File: beginner/java/1074.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {

        Locale.setDefault(Locale.US);

        Scanner scanner = new Scanner(System.in);

        int n, i, x;

        n = scanner.nextInt();

        for(i = 0; i < n; i++){
            x = scanner.nextInt();

            if(x == 0)
                System.out.println("NULL");
            else if(x > 0 && x % 2 == 0)
                System.out.println("EVEN POSITIVE");
            else if(x > 0 && x % 2 != 0)
                System.out.println("ODD POSITIVE");
            else if(x < 0 && x % 2 == 0)
                System.out.println("EVEN NEGATIVE");
            else if(x < 0 && x % 2 != 0)
                System.out.println("ODD NEGATIVE");
        }

        scanner.close();

    }
}

